int8 mask_num
fcsc/PointCloud2Array sandwich_pcl
---
geometry_msgs/PoseArray sandwich_pose
